src/markdown-pages/add-ons/rook.md

```diff
@@ -30,8 +30,10 @@ flags-table
 
 ## Block Storage
 
-For Rook versions 1.4.3 and later, block storage is required.
-For Rook versions earlier than 1.4.3, block storage is recommended in production clusters. For disk requirements, see [Add-on Directory Disk Space Requirements](/docs/install-with-kurl/system-requirements/#add-on-directory-disk-space-requirements).
+Rook versions 1.4.3 and later require a dedicated block device attached to each node in the cluster.
+To meet this block storage requirement, you must add an unformatted disk that is used only for Rook to each node.
+For Rook versions earlier than 1.4.3, a dedicated block device is recommended in production clusters.
```

Comment:
```suggestion
**NOTE** If you must to use old versions (not recommended), be aware that for versions earlier than `1.4.3`, a dedicated block device is recommended in production clusters.
```

Should we not encourage use the latest ones?

Reply:
@camilamacedo86 that is also a great point. If we want to encourage users to be on version 1.4.3 or later of Rook, then we can definitely add that as a separate recommendation. The recommendation to use version 1.4.3 could be added to the top of the topic, since it applies to more than just this block storage section. Adding it to the top would also make it easier for users to see. What do you think?

Reply:
@camilamacedo86 just a heads up that I added this separate docs story to update this topic to make it clear that version 1.4.3 or later is recommended: https://app.shortcut.com/replicated/story/65464/add-info-that-1-4-3-or-later-of-rook-is-recommended

Going to merge the PR, then add the version recommendation as a separate PR. Thanks again for your feedback!